### ISSUE TITLE
PostgreSQL container logs not displayed

### ISSUE THREAD

**Issue by Tset-Noitamotua** [jesseduffield/lazydocker#170]
**Describe the bug**
Logs of a customized PostgreSQL container I use are not displayed

**To Reproduce**
1. docker run --name ehrdb -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=postgres -d -p 5432:5432 rippleosi/ethercis-database-docker
2. start lazydocker as container
3. check Logs of ehrdb container - they are empy

**Expected behaviour**
Expected to see same logs that I get from following command
```bash
> docker container logs ehrdb 

# example output
2019-09-11 06:27:48.178 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2019-09-11 06:27:48.178 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2019-09-11 06:27:48.180 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2019-09-11 06:27:48.191 UTC [18] LOG:  database system was shut down at 2018-10-24 08:54:17 UTC
2019-09-11 06:27:48.193 UTC [1] LOG:  database system is ready to accept connections
```

**Screenshots**
![image](https://user-images.githubusercontent.com/5635462/64673233-e29d6380-d46d-11e9-8171-4415cb5a3bfe.png)

**Desktop (please complete the following information):**
 - OS: Linux (Deepin 15.11 - kernel 5.0)
 - Lazydocker Version v0.7.4

**Additional context**
I Also tried starting ehrd container from a customcommand for images from inside lazydocker, same behavior as described above.

**Comment by mjarkk** [comment]
Does this happen with other contains too?

**Comment by dawidd6** [comment]
Running lazydocker in Docker container won't give you logs or CPU usage. Known bug.

**Comment by mjarkk** [comment]
@dawidd6 Is this just impossible due to files outside of the docker container?  
Because if so we could better just remove this `logs` tab from the docker container.

**Comment by dawidd6** [comment]
Honestly I have no idea why it does not work in Docker. We have `docker` binary in image and yet it refuses to show logs, but other stuff is working.

**Comment by mjarkk** [comment]
Ow i see this is indeed wired,  
when i run docker:dind with the docker sock mounted it shows the logs just fine.
```sh
docker run \
  -v /var/run/docker.sock:/var/run/docker.sock \
  -it --rm docker:dind sh

/ # docker logs redis
```

**Comment by dawidd6** [comment]
Yep, `docker` will show logs, but `lazydocker` not. Guess it becomes lazier in Docker.

**Comment by Tset-Noitamotua** [comment]
> Running lazydocker in Docker container won't give you logs or CPU usage. Known bug.

confirm, running "normally" installed lazydocker displays logs as expected

![image](https://user-images.githubusercontent.com/5635462/64686703-5f8a0680-d489-11e9-9b70-f170cac79ada.png)

**Comment by dawidd6** [comment]
Added a notice about this: https://github.com/jesseduffield/lazydocker/commit/ef1b2b57b0e17b1635421cc40dfa54f2acf788e6

**Comment by facorazza** [comment]
Any news about why this happens?

**Comment by jschneekloth** [comment]
Any progress on this front?  This is sort of a blocker for docker installations.

**Comment by DavidAskwith** [comment]
@jesseduffield I did some investigation into this issue. I was able to reproduce this with the latest `lazyteam/lazydocker`  docker hub image. 

I set up a development environment with the latest version of the code base. The issue was not reproducible with images built from the current version of the code base. I rolled the repository back to the point in time the `lazyteam/lazydocker`  image was released and I was able to reproduce again.

I think this can be closed by pushing a new version of the image.
